docs/dev_guide/migration/migration-workflow.rst

```diff
@@ -354,6 +369,20 @@ in the Codeplay plugin documentation:
 * `Install the oneAPI for AMD GPUs plugin <https://developer.codeplay.com/products/oneapi/amd/guides/>`_ from Codeplay.
 * `Install the oneAPI for NVIDIA GPUs plugin <https://developer.codeplay.com/products/oneapi/nvidia/guides/>`_ from Codeplay.
 
+Stage 5: Validate the New SYCL Application
+------------------------------------------
+
+After you have built your converted code, validate your new SYCL application to
+check for correct functionality after migration.
+
+Use CodePIN to Validate Migrated Code
+*************************************
+
+Use the CodePIN signature (captured in Stage 1) to validate your migrated code.

```

Comment:
If the CodePin feature has been enabled during the migration time,
project signature will be logged during the execution time.
The signature contains the data value of each execution checkpoint, which can be verified manually or with an auto-analysis tool.